### PR TITLE
export: adds disable class to download button

### LINKED ISSUE #539: Disable Download button when no checkbox selected
To avoid confusion, the Download button should be disabled when no checkbox is selected

screen shot 2015-11-20 at 6 14 08 pm

### PR DESCRIPTION
**Pull request by ioannistsanaktsidis**
* Closes #539.

Signed-off-by: Ioannis Tsanaktsidis <ioannis.tsanaktsidis@cern.ch>

### CODE REVIEW

File: inspire/base/static/js/search/search_results.js [hunk 1]
```diff
@@ -216,7 +218,11 @@ define(
           }
         }
         if (sList.length != 0) {
+          console.log(sList);
```
Comment (jmartinm) [inline inspire/base/static/js/search/search_results.js]: This shouldnt be here?

**Comment by ioannistsanaktsidis** [comment]
Updated!

**Comment by jmartinm** [comment]
The disabling works but I see a problem I have not seen in the past. When using the select-all checkbox

screen shot 2015-11-23 at 11 46 55 am

The `Select all xx results` link does not appear. Maybe related to this PR?

**Comment by ioannistsanaktsidis** [comment]
This is what I am fixing now. It will be handled to the next PR. It is caused from the results per page. I will make a PR soon.

**Comment by jmartinm** [comment]
ok thanks

**Pull request merged**